.gitignore

```diff
@@ -34,4 +34,5 @@
 *.tdo
 _minted*
 .texpadtmp/
+.idea
```

Comment:
1. We only want to exclude the folder on the root level. So this should have a leading slash.
2. We only want to exclude the folder, not any files with the same name (just so that it's explicit). So we should have a trailing slash.

So can you please change this to /.idea/ ? Thanks!

Reply:
Ok. Sorry. Hope it is correct now.